third_party/2and3/attr/__init__.pyi

```diff
@@ -0,0 +1,268 @@
+from typing import Any, Callable, Dict, Generic, List, Optional, Sequence, Mapping, Tuple, Type, TypeVar, Union, overload
+# `import X as X` is required to make these public
+from . import exceptions as exceptions
+from . import filters as filters
+from . import converters as converters
+from . import validators as validators
+
+_T = TypeVar('_T')
+_C = TypeVar('_C', bound=type)
+
+_ValidatorType = Callable[[Any, Attribute, _T], Any]
+_ConverterType = Callable[[Any], _T]
+_FilterType = Callable[[Attribute, Any], bool]
+_ValidatorArgType = Union[_ValidatorType[_T], List[_ValidatorType[_T]], Tuple[_ValidatorType[_T], ...]]
```

Comment:
Don't have time to read through the attrs code right now, but why isn't this just Iterable or Sequence? Does it really accept only lists and tuples?

Reply:
yes

Reply:
The code literally does `isinstance(validator, (list, tuple)):`

Reply:
That's unfortunate because list and tuple are invariant, so subtypes of `_ValidatorType` may end up not getting accepted.

Reply:
Good point.  Confirmed that I'm getting errors with lists of validators.  I'm going to use `Sequence` for now and I'll open a separate ticket against attrs to see if this can be addressed in the code.